data/mods/Magiclysm/Spells/magus.json

```diff
@@ -183,9 +183,11 @@
     "shape": "blast",
     "spell_class": "MAGUS",
     "energy_source": "MANA",
-    "difficulty": 4,
-    "base_casting_time": 200,
-    "base_energy_cost": 300,
+    "difficulty": 9,
```

Comment:
kk i thought it still overpovered, but it has only 2 minute length, so its fine

Reply:
press "resolved" on this review, i forgot it doesnt work as a comment field

Reply:
i think it still way to good, even if it's just 2 minutes, you become an absolute killing machine, even just been able to outwalk everything in the game is very strong